rdr_service/dao/consent_dao.py

```diff
@@ -13,6 +13,10 @@ class ConsentDao(BaseDao):
     def __init__(self):
         super(ConsentDao, self).__init__(ConsentFile)
 
+    def get(self, obj_id) -> ConsentFile:
+        with self.session() as session:
+            return session.query(ConsentFile).filter(ConsentFile.id == obj_id).one_or_none()
+
```

Comment:
Doesn't the `get` method on the parent class provide this already?

Reply:
Good catch! I'm not really sure why I did it this way. But you're right, BaseDao's `get` gives the exact same thing. I'll update it.